docs/status.md

```diff
@@ -46,7 +46,7 @@ one of the following tracking labels.
 | add                      | yes           | yes          | yes            | yes             | yes         |
 | after_all                | yes           | yes          | yes            | yes             | no          |
 | all_gather               | yes           | revisit      | no             | no              | no          |
-| all_reduce               | yes           | revisit      | yes            | no              | no          |
+| all_reduce               | yes           | yes          | revisit        | no              | no          |
```

Comment:
https://github.com/openxla/stablehlo/blob/01bbf2ae747f0ab95347d368368b44fefd34fb08/stablehlo/dialect/StablehloOps.cpp#L294

The verification is `revisit` based on https://github.com/openxla/stablehlo/issues/498

Reply:
+1

Reply:
Makes sense, thanks for the pointing to the related line @sdasgup3